src/energyplus/ForwardTranslator/ForwardTranslateZoneHVACPackagedTerminalHeatPump.cpp

```diff
@@ -317,6 +320,10 @@ namespace energyplus {
     } else if (boost::optional<CoilCoolingDXVariableSpeed> dxCoil = coolingCoil.optionalCast<CoilCoolingDXVariableSpeed>()) {
       _coolingCoil = translateCoilCoolingDXVariableSpeedWithoutUnitary(dxCoil.get());
 
+      m_map.insert(std::make_pair(coolingCoil.handle(), _coolingCoil.get()));
+    } else if (boost::optional<CoilCoolingDX> dxCoil = coolingCoil.optionalCast<CoilCoolingDX>()) {
+      _coolingCoil = translateCoilCoolingDXWithoutUnitary(dxCoil.get());
```

Comment:
Follow Coil:Cooling:DX:SingleSpeed and Coil:Cooling:DX:VariableSpeed as a guide.